src/App/Pages/ContactPage.elm

```diff
@@ -10,8 +10,18 @@ import Material.Options as Options
 import Material.Button as Button
 
 
-view : Model -> Html Msg
-view model =
+init : ContactForm
+init =
+    { name = ""
+    , email = ""
+    , telephone = ""
+    , subject = ""
+    , message = ""
+    }
+
+
+view : Model -> ContactForm -> Html Msg
```

Comment:
Well, `ContactPage.View` should call `Msg`s defined in `ContactPage.Model`, which will have a corresponding `update` function in `ContactPage.Update`.

btw, you have https://github.com/etaque/elm-form that I like working with when I have many fields.